### DISCUSSION TITLE
How to delete unnecessary folders

### DISCUSSION THREAD

**Discussion by alisa0920** [requarks/wiki#7278] [Help / Questions]
hello everyone:
  i do not know how to delete unnecessary folders, just like this picture,i want delete some folder but i can not find any  button to delete it

**Comment by alisa0920** [comment]
![1722077323400](https://github.com/user-attachments/assets/50e99597-5e54-4593-b501-dd9626c7ab96)

**Reply by alisa0920** [reply]
i really need help! thanks

**Comment by akib1689** [comment]
I don't think the current stable version supports this assets folder management feature. You can find this is the official documentation [here](https://docs.requarks.io/guide/assets)

See the manage asset folder section of this page

**Comment by FreeSynergy** [comment]
The managing of assets is very important, to have a stable version of the Wiki …

Any idea, how we can work on this? (I had hoped that the new version would solve the problem. But hardly anything is being done. (I'm not giving up yet).)

Nevertheless, this function should work because it creates a lot of folders that cannot be deleted or moved.
